File: isma-next-hsm/src/main/kotlin/ru/nstu/isma/hsm/exp/EXPMathFunction.java
package ru.nstu.isma.hsm.exp;

import ru.nstu.isma.hsm.var.HMVariable;

import java.io.Serializable;

/**
 * Created by Bessonov Alex
 * on 24.03.2015.
 */
public class EXPMathFunction extends EXPFunctionOperand implements Serializable {
    public EXPMathFunction(HMVariable var) {
        super(var.getCode());
        this.variable = var;
    }
}

File: isma-next-hsm/src/main/kotlin/ru/nstu/isma/hsm/exp/EXPMathFunction.kt
package ru.nstu.isma.hsm.exp

import ru.nstu.isma.hsm.`var`.HMVariable
import java.io.Serializable

/**
 * Created by Bessonov Alex
 * on 24.03.2015.
 */
class EXPMathFunction(`var`: HMVariable) : EXPFunctionOperand(`var`.code), Serializable {
    init {
        variable = `var`
    }
}